README.md

```diff
@@ -72,6 +72,49 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-check/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+### What features does `cargo-semver-checks` enable in the tested crates?
+
+By default, crates are built with all features excluding heuristically selected features that usually break semver. More precisely, features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__` are disabled.
+
+The flags below can be used to select the checked features set:
+```
+--default-features
+    Use only the crate-defined default features, as well as any features added explicitly via other flags.
+
+    Using this flag disables the heuristic that enables all features except `unstable`, `nightly`, `bench`, `no_std`, and ones starting with `__`.
+
+--only-explicit-features
+    Use no features except ones explicitly added by other flags.
+
+    Using this flag disables the heuristic that enables all features except `unstable`, `nightly`, `bench`, `no_std`, and ones starting with `__`.
+
+--features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in both the baseline and the current version of the crate
+
+--baseline-features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in the baseline version of the crate only
+
+--current-features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in the current version of the crate only
+
+--all-features
+    Use all the features, including features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__`, that are otherwise disabled by default
+```
```

Comment:
This code block doesn't look good when rendered:
https://github.com/staniewzki/cargo-semver-check/tree/cli-specify-features#what-features-does-cargo-semver-checks-enable-in-the-tested-crates

Reply:
Yes, it is little bit awkward. Some solutions I see:
1. Try to wrap the lines to make them shorter. This still results in a big code block.
2. Move it out of a codeblock, and briefly explain all the flags related to features.
3. Scrap it altogether, and explain the functionality in a shorter way.

Reply:
I implemented the 2. solution, and I think I like it.